makefile

```diff
@@ -23,23 +23,27 @@ filenamesc := ft_strlen.c \
 			ft_strncmp.c \
 			ft_atoi.c \
 
-filenameso := $(filenamesc:%.c=%.o) 
-CC = gcc -Wall -Wextra -Werror
+filenameso := $(filenamesc:%.c=%.o)
+
+CC = gcc
+CFLAGS = -Wall -Wextra -Werror
+NAME = libft.a
```

Comment:
It does, but it also makes it a lot easier for you to reuse your Makefile for something else afterwards.